debezium-server/debezium-server-redis/src/main/java/io/debezium/server/redis/RedisStreamChangeConsumer.java

```diff
@@ -129,9 +157,13 @@ public void handleBatch(List<ChangeEvent<Object, Object>> records,
             throws InterruptedException {
         DelayStrategy delayStrategy = DelayStrategy.exponential(initialRetryDelay, maxRetryDelay);
 
-        LOGGER.info("Handling a batch of {} records", records.size());
+        LOGGER.trace("Handling a batch of {} records", records.size());
         batches(records, batchSize).forEach(batch -> {
             boolean completedSuccessfully = false;
+            if (relativeMinIdTrim != null && maxLenTrim == null) {
+                long minIdTrim = System.currentTimeMillis() - relativeMinIdTrim;
+                addParams.minId(minIdTrim + "");
```

Comment:
```suggestion
                addParams.minId(String.valueOf(minIdTrim));
```

Reply:
I read that "" + x performs a lot fast in all Java version, so I went with that. But I can change.